spec/factories/notifications.rb

```diff
@@ -0,0 +1,23 @@
+FactoryBot.define do
+  factory :notification do
+    notification_type { Notification::NOTIFICATION_TYPES.sample }
```

Comment:
can't be sample if by default u have `association :notificable, factory: :user` !!!